### DISCUSSION TITLE
buddhism/2024/mind_ground/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#123] [General]
# buddhism/2024/mind_ground/

https://shanenull.com/buddhism/2024/mind_ground/

**Comment by shane0** [comment]
# mind ground

- https://shanenull.com/buddhism/2025/html/mindground.html
- no fear no thought

## 3 links of cultivation

- [study Dharma teachings](https://shanenull.com/buddhism/2024/books/)
- meritorious service
- [meditation](https://shanenull.com/buddhism/2024/mind_ground/)

```text
法無
```

> "I CANNOT ESCAPE DEATH. BUT AT LEAST I CAN ESCAPE THE FEAR OF IT." - EPICTETUS

**Reply by shane0** [reply]
https://shanenull.com/buddhism/latest/mind_ground/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/four_abodes/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/three_jewels/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/four_abodes/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2021/docs_2020/books/rigdzin/
https://shanenull.com/buddhism/2021/docs_2020/dzogchen/vajra-essence/
https://shanenull.com/buddhism/2021/docs_2021/sadhana/sunset/
https://shanenull.com/buddhism/2023/shunryu/#beginners-mind
https://shanenull.com/buddhism/2023/extremes/
https://shanenull.com/buddhism/2023/leaks/#leaks

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2022/state/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/evening_ritual/

**Reply by shane0** [reply]
pronunciation for 法無 ("fǎ wú") in Mandarin Chinese is as follows:

法 (fǎ): pronounced with a falling-rising tone, similar to "fah" with a rising inflection.
無 (wú): pronounced with a rising tone, similar to "woo" but with a tone that rises slightly toward the end.

**Reply by shane0** [reply]
法 fǎ 無 wú https://www.purpleculture.net/chinese-english-dictionary/?word=%E6%B3%95%E7%84%A1

**Comment by shane0** [comment]
https://github.com/shane0/buddhism/discussions/117

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/four_abodes/

**Reply by shane0** [reply]
# definition of abide

- abide nowhere
- abide in nothing

>> accept or act in accordance with (a rule, decision, or recommendation).
>> be able to tolerate (someone or something).

**Comment by shane0** [comment]
# all experiences in meditation are illusory and insubstantial

- https://youtu.be/IIlAOUyINXw

**Comment by shane0** [comment]
# temporary workarounds

It is quite a challenge to maintain pure awareness and constantly abide in the state of no-thought.

If we feel sleepy, we may switch to the method of breath counting to awaken ourselves again.

If our mind is scattered, we may observe our 
breath in order to bring our mind back.

If our mind is clouded, we may contemplate on the impermanent, empty nature of phenomena.

If we feel bored or dull, we may look at the buddha statue and use his example of perfect compassion, wisdom, and vows to motivate us to be diligent.

These are temporary antidotes to the problems we encounter in meditation.

**Comment by shane0** [comment]
# heart sutra + mind ground

- the heart sutra and diamond sutra prepend everything
- extends the keys and seals and marks of existence
- https://github.com/shane0/buddhism/issues/104

**Reply by shane0** [reply]
- consciousnesses https://youtu.be/2w8SVayCess
- https://shanenull.com/buddhism/2023/awareness/
